includes/classes/PluginSupport/ScriptedInstallHelpers.php

```diff
@@ -173,7 +181,7 @@ public function addConfigurationGroup(string $group_title, array $properties): i
 
         if (empty($sql_data_array['sort_order'])) {
             // manually set sort order if none provided:
-            zen_db_perform(TABLE_CONFIGURATION_GROUP, ['sort_order' => $insert_id], 'update', ' WHERE configuration_group_id = ' . $insert_id);
+            zen_db_perform(TABLE_CONFIGURATION_GROUP, ['sort_order' => $insert_id], 'update', ' configuration_group_id = ' . $insert_id);
```

Comment:
I thought that one of the 'tenents' of the ScriptedInstall classes was to use its defined `$this->dbConn` so that any MySQL errors within a zc_plugin were recoverable.

Reply:
As far as bespoke SQL queries run by a plugin's own db calls, I agree.

For things that internal core should handle itself, I opted to use internal tooling like `zen_db_perform()` instead of writing longer duplicated code which just does the same thing.